uk.co.compucorp.civicrm.tasksassignments/views/contact/documents.html

```diff
@@ -47,20 +47,43 @@
         <table class="table table-striped {{prefix}}table-documents">
           <thead>
             <tr>
-              <th>Type</th>
-              <th>Contact</th>
-              <th ng-if="settings.extEnabled.assignments">Assignment</th>
-              <th>Due Date</th>
-              <th>Expiry Date</th>
-              <th>Status</th>
+              <th ng-click="sortBy('type')">
```

Comment:
It looks like a lot of code repetition to me. Could this be condensed by looping through an array of objects that contain at least the following props:

```js
[
  {
    name: 'type',
    label: 'Type'
  }
]
```
?

Reply:
Well that's a good idea. As for me since we have condition in `<th ng-click="sortBy('case_id')" ng-if="settings.extEnabled.assignments">`, I think we will have problem in this case. So i prefer to stick with this solution.

Reply:
You can add an additional prop `condition` or smth like that?